contents/handbook/people/spending-money.md

```diff
@@ -76,7 +76,7 @@ We'd prefer you to use a laptop. This is so when we host meetups in real life, y
 * For engineering roles, we recommend a Macbook Pro 14-inch M4 Pro, with the 12-core CPU, 16-core GPU upgrade and 48GB of RAM. By default grab the smallest SSD (512gb), but you can upgrade if you need to.
 * For all other roles, we currently recommend a Macbook Air with an Apple Silicon (M2) processor and 16GB of RAM.
 
-These are just general guidelines - the most important thing is that you select the model that is appropriate for _your_ needs. If your requirements are different to the guidelines above please just ask Fraser.
+These are just general guidelines - the most important thing is that you select the model that is appropriate for _your_ needs. If your requirements are different to the guidelines above please just ask Fraser. Please note that when you are ordering machines internationally (eg not in US or UK or if you are in a place where Macs are more expensive) it sometimes is quicker & significantly cheaper to get your laptop delivered to the location of your in-person onboarding and you may save significant money/be able to get an upgraded machine for less, this is worth it if you could save 20%+ . You can co-ordindate with your onboarding buddy to recceive delivery of this (this is very true if your onboarding is in the UK or US as we hold machines in stock to make it easier).
```

Comment:
```suggestion
These are just general guidelines - the most important thing is that you select the model that is appropriate for _your_ needs. If you are ordering outside of the US or UK, it may be quicker & significantly cheaper to get your laptop delivered to the location of your in-person onboarding, as we keep a small stock. Check with your onboarding buddy if this is the case. 
```